### PR TITLE
add sleep after publishing moveit scene msg

### PR DESCRIPTION
**Pull request by knorth55**
according to moveit tutorials, we need to add sleep after publishing scenes.

http://docs.ros.org/indigo/api/pr2_moveit_tutorials/html/planning/src/doc/planning_scene_ros_api_tutorial.html

### CODE REVIEW

File: jsk_arc2017_baxter/euslisp/lib/arc-interface.l [hunk 1]
```diff
@@ -659,20 +661,24 @@
...
             :relative-pose (send cube :copy-worldcoords)
-            :object-id (format nil "~A_tote" (arm2str arm)))))
+            :object-id (format nil "~A_tote" (arm2str arm)))
+      (unix::usleep (* 500 1000))))
   (:delete-tote-scene (arm)
-    (send *co* :delete-object (gethash arm tote-cubes-)))
+    (send *co* :delete-object (gethash arm tote-cubes-))
+    (unix::usleep (* 500 1000)))
   (:wipe-all-scene ()
     (send *co* :wipe-all))
```
Comment (pazeshun) [inline jsk_arc2017_baxter/euslisp/lib/arc-interface.l]: Don't we need sleep here?

Comment (knorth55) [inline jsk_arc2017_baxter/euslisp/lib/arc-interface.l]: we publish msg in `:delete-object`.

Comment (pazeshun) [inline jsk_arc2017_baxter/euslisp/lib/arc-interface.l]: I think `:wipe-all` also publish msg.
https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/master/pr2eus_moveit/euslisp/collision-object-publisher.l#L182

Comment (knorth55) [inline jsk_arc2017_baxter/euslisp/lib/arc-interface.l]: https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/315

Comment (pazeshun) [inline jsk_arc2017_baxter/euslisp/lib/arc-interface.l]: Understood.

**Pull request merged**